src/libraries/System.Net.Http/src/System/Net/Http/HttpRequestMessage.cs

```diff
@@ -189,7 +189,11 @@ private void InitializeValues(HttpMethod method, Uri? requestUri)
             }
             if ((requestUri != null) && (requestUri.IsAbsoluteUri) && (!HttpUtilities.IsHttpUri(requestUri)))
```

Comment:
`HttpUtilities.IsHttpUri(requestUri)` - we return true now if `scheme==blob`. What happens if we have a url like `blob:not-http://foo/bar`? Should `false` be returned in that case?

Reply:
No as this is created from the javascript environment normally with URL.createURLObject.  The browser environment in this case will handle the error if something went wrong.